examples/instrumentation/configure_instrumentation_global.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace _;
+
+use OpenTelemetry\Example\Example;
+use const PHP_EOL;
+
+/**
+ * This example uses SPI (see root composer.json extra.spi) to configure an example auto-instrumentation from a YAML file
+ */
+// EXAMPLE_INSTRUMENTATION_SPAN_NAME=test1234 php examples/instrumentation/configure_instrumentation.php
+putenv('OTEL_PHP_AUTOLOAD_ENABLED=true');
+putenv(sprintf('OTEL_EXPERIMENTAL_CONFIG_FILE=%s/%s', __DIR__, 'otel-sdk.yaml'));
```

Comment:
Might make sense to add `getcwd()` as path to the configuration file locator to allow loading relative paths https://github.com/open-telemetry/opentelemetry-php/blob/500f6ee1bd32a2fd289125aecd0d38a123c57596/src/Config/SDK/Configuration/ConfigurationFactory.php#L101

Reply:
Agreed. Done.